Compiler/src/stmtinfo.jl

```diff
@@ -491,7 +491,6 @@ perform such accesses.
 struct GlobalAccessInfo <: CallInfo
     bpart::Core.BindingPartition
 end
-GlobalAccessInfo(::Nothing) = NoCallInfo()
```

Comment:
This one is required

Reply:
I’ve checked all the callsites, and `bpart` was always `Core.BindingPartition`. Is this `GlobalAccessInfo(::Nothing)` intended for future use?

Reply:
Never mind, we have the `saw_latestworld` branch.